sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -324,7 +324,9 @@ private[joins] object UnsafeHashedRelation {
  *
  * see http://java-performance.info/implementing-world-fastest-java-int-to-int-hash-map/
  */
-private[execution] final class LongToUnsafeRowMap(var mm: TaskMemoryManager, capacity: Int)
+private[execution] final class LongToUnsafeRowMap(
+    @transient val mm: TaskMemoryManager,
```

Comment:
In generally, this type of 'fix' is actually wrong. If the object needs a reference to this thing, then it won't work wherever it's deserialized. In some cases, that happens to work out because the object is only accidentally serialized and never used, or doesn't actually touch the transient field after it's serialized. Is one of those the case here?